lib/userAuthenticator.js

```diff
@@ -157,7 +157,7 @@ UserAuthenticator.prototype._authorizationFlow =  function (options) {
                     _done = true;
                     res.writeHead(200, { 'Content-Type': 'text/plain' });
                     if (error) {
-                        res.end(`Authentication failed:\n\n${error.message}`);
+                        res.end(error.message);
```

Comment:
All errors were created earlier in this code path within the cli and already are prefixed with 'Authentication failed'... so the error message on the browser also appeared as:

```
Authentication failed:

Authentication failed: <real error message>
```